consensus/src/experimental/buffer_item.rs

```diff
@@ -31,39 +32,82 @@ fn generate_commit_proof(
     )
 }
 
-fn aggregate_ledger_info(
+fn verify_signatures(
+    unverified_signatures: &PartialSignatures,
```

Comment:
this should take in ownership instead of reference and use into_iter

Reply:
Modified to take ownership - but `into_iter` is not needed as this is an iterator on references.